geonode/static/lib/js/jquery.tree-multiselect.min.js

```diff
@@ -1,2 +1,2 @@
\ No newline at end of file
+/* jQuery Tree Multiselect v2.2.1 | (c) Patrick Tsai | MIT Licensed */
```

Comment:
It looks like somehow you got a new version of jQuery... this should not be handled like this. There are bower and grunt projects on geonode static for that